docs/source/guide/zne-2-use-case.myst

```diff
@@ -0,0 +1,43 @@
+---
+jupytext:
+  text_representation:
+    extension: .myst
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.10.3
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# When should I use ZNE?
+ 
+## Advantages 
+
+ZNE generally works well for circuits of low depth and/or those exhibiting noise 
+that can be described by a simple model 
+[[32](https://mitiq.readthedocs.io/en/latest/zz_bibliography.html)]. 
+The main advantage of ZNE is that the technique can be applied without knowledge of 
+the undelying noise model. Therefore it can be a good option in situations where 
+tomography is impractical.
+  
+
+## Disadvantages 
+
+In some instances the results of the extrapolation can exhibit a large bias
+[[33](https://mitiq.readthedocs.io/en/latest/zz_bibliography.html)].
```

Comment:
I am not sure what paper you want to cite here.
The proper way to add a citation is: {cite}`label_of_a_reference_in_references.bib_file` . For example:
https://github.com/unitaryfund/mitiq/blob/f7bd85bd0daf4b4d403c27f96ff58c784629431d/docs/source/guide/pec-1-intro.myst#L132

Reply:
@Misty-W I guess you are referring here to etiher {cite}Lowe_2021_PRR or {cite}PhysRevA.104.052607.

Reply:
Yes, I changed it back to {cite}PhysRevA.104.052607`. I was trying a workaround to make the link work in Jupyter notebook, but that's probably a minor concern compared to maintainability of the references.